ASM/c/model_text.c

```diff
@@ -0,0 +1,33 @@
+#include "model_text.h"
+
+uint16_t illegal_model = 0;
+
+void draw_illegal_model_text(z64_disp_buf_t *db) {
+
+    // Only draw when paused
+    if (!(illegal_model && z64_game.pause_ctxt.state == 6)) {
+        return;
+    }
+    
+    // Setup draw location
+    int str_len = 36;
+    int total_w = str_len * font_sprite.tile_w + triforce_sprite.tile_w;
```

Comment:
```suggestion
    int total_w = str_len * font_sprite.tile_w;
```

Reply:
Oops, you can tell where I copied this code from, lol